cpu/kinetis/periph/uart.c

```diff
@@ -296,7 +305,19 @@ static inline void uart_init_lpuart(uart_t uart, uint32_t baudrate)
     LPUART_Type *dev = uart_config[uart].dev;
     uint32_t clk = uart_config[uart].freq;
 
-    /* Remember to select a module clock in board_init! (SIM->SOPT2[LPUART0SRC]) */
+    /* Set LPUART clock source */
+#ifdef SIM_SOPT2_LPUART0SRC
```

Comment:
The value of this definition doesn't matter? It only needs to be defined to execute the code below? It's just because above there is:

```c
#define SIM_SOPT2_LPUART0SRC SIM_SOPT2_LPUARTSRC
```

thus the value might be needed or evaluated.

Reply:
They are defined by the vendor and put headers for the CPUs which have the appropriate hardware module. The definition in cpu_conf_kinetis.h is only a compatibility definition for some CPU headers which have named the bit field LPUARTSRC instead of LPUART0SRC. See the other definitions in the same area in cpu_conf_kinetis.h.

Reply:
Ok I got it. Thanks!